apps/api-server/src/models/Status.js

```diff
@@ -0,0 +1,130 @@
+
+const sanitize = require('../util/sanitize');
+const config = require('config');
+const getExtraDataConfig = require('../lib/sequelize-authorization/lib/getExtraDataConfig');
+const userHasRole = require('../lib/sequelize-authorization/lib/hasRole');
+const seqnr = require('./lib/seqnr');
+
+module.exports = function (db, sequelize, DataTypes) {
+  let Status = sequelize.define(
+    'statuses',
+    {
+      projectId: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+      },
```

Comment:
Deze projectId hoeft hier tocht niet omdat dit gebruikt wordt in een association? Als het goed is als je een belongsTo gebruikt, dan wordt er een foreign key voor je aangemaakt met de juiste verwijzing naar de bijbehorende parent (project) entry. Weet niet zeker of dat op deze manier gedaan wordt.

https://sequelize.org/docs/v6/core-concepts/assocs/#implementation

Reply:
Dat zou ook werken inderdaad. Ik vind het fijn om het toch expliciet toe te voegen omdat ik in deze lijst kijk als ik wil weten welke velden een object heeft, en niet in de associations.

Ook dat is hoop ik consistent zo in de hele API.